docs/source/app_development/index.rst

```diff
@@ -15,8 +15,10 @@ Overview of the separate in-depth sections in this part of the documentation:
 
 To get an idea of how proper apps may be built and used, you can check out these examples:
 
-.. toctree::
-    :maxdepth: 1
+.. note::
+
+    AiiDAlab expects that app source code is managed with `git <https://git-scm.com/>`__.
+    To start tracking the app development with git, open the terminal once more, and execute:
```

Comment:
I am not sure what was the idea behind it, but this broke the docs.

Reply:
Looks like some weird merge conflict.

Reply:
There was a merge conflict, whereby the pre-commit now removes trailing whitespace (that was present on line 19). Wouldn't have thought this should break the directive though 🤷

Reply:
Should be fixed in #166 .